### ISSUE TITLE
Creates a terminal experience - use the CLI

### ISSUE THREAD

**Issue by roselmamendes** [roselmamendes/pergunte-stackoverflow-cli#2]
Build how gonna be the interaction using the CLI.

`python pergunte_stackoverflow "dependencies golang"`

**Output**

```bash
Endpoint: <>
Parameters: q -> <> ; sort -> relevance ; site -> stackoverflow ; order -> desc

=========================================================
Issues with goLang dependencies

Answer with higher score:
For your first question, it's because GL.h is missing from your ubuntu machine. You could get it installed by installing the dev lib as stated as requirement in their <a href=\"https://github.com/go-gl/gl\" rel=\"nofollow\">go-gl github page</a>:

<pre><code>sudo apt-get install libgl1-mesa-dev\n</code></pre>


After this, you could go get the core lib. I've tested on my amazon ubuntu instance and worked fine.


If there still are something missing, try google using the key word:

<pre><code>ubuntu the_missing_file_name is missing\n</code></pre>


Also, I find this page very useful for your installing issues: <a href=\"https://github.com/google/gxui/wiki/Installation\" rel=\"nofollow\">https://github.com/google/gxui/wiki/Installation</a>


Basically, install these packages:

\n\n<pre><code>sudo apt-get install libgl1-mesa-dev (or freeglut3-dev)\nsudo apt-get install libxrandr-dev\nsudo apt-get install libxcursor-dev\nsudo apt-get install libxi-dev\nsudo apt-get install libxinerama-dev\n</code></pre>


After you installed all the required dev packages, you could go get glfw successfully :)
```
